src/client/method-status.c

```diff
@@ -208,10 +208,17 @@ static void print_unit_info(unit_info_t *unit_info, size_t name_col_width) {
         fprintf(stdout, "%s", unit_info->id);
         name_col_width -= unit_info->id ? strlen(unit_info->id) : 0;
         name_col_width += PRINT_TAB_SIZE;
-        for (i = PRINT_TAB_SIZE + name_col_width; i > PRINT_TAB_SIZE; i -= PRINT_TAB_SIZE) {
+        i = name_col_width;
+        while (i >= PRINT_TAB_SIZE) {
                 fprintf(stdout, "\t");
+                i -= PRINT_TAB_SIZE;
+        }
+        while (i > PRINT_TAB_SIZE) {
+                fprintf(stdout, " ");
+                i--;
```

Comment:
I'm just wondering if it wouldn't be better to extract table printing from https://github.com/eclipse-bluechi/bluechi/blob/main/src/client/method-list-units.c#L175 and reuse it also here. Calculate column lengths and using formatting string seems to me much easier to understand than this tab+space alignment

Reply:
But that can be done separately in a different PR

Reply:
If I will print only spaces without the tabs.
Will it be more understandable?

Reply:
Its probably fine as it is right now. 
As Martin wrote, we could optimize and reuse other code here, but thats a different task. Lets tackle it in a new PR. Could you create a GH issue for it if we want to track it? @mwperina